### PR TITLE
[IMP] accounting: different ways to record a payment

### PR DESCRIPTION
**Pull request by Donapi**
[ID: 3092721](https://www.odoo.com/web#cids=1&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form&id=3092721)

### CODE REVIEW

File: content/applications/finance/accounting/receivables/customer_payments/recording.rst [hunk 1]
```diff
@@ -74,11 +79,12 @@ statement.
...
 -------------
 
-Batch payments allow you to group different payments to ease reconciliation. They are also useful
-when you deposit checks to the bank or for SEPA Payments.
-Go to :menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
-Vendors --> Batch Payments`. In the list view of payments, you can select several payments and group
-them in a batch clicking on :menuselection:`Action > Create Batch Payment`.
+Batch payments allow you to group different payments to ease :doc:`reconciliation
+<../../bank/reconciliation/use_cases>`. They are also useful when you deposit checks to the bank or
+for SEPA Payments. To do so go to :menuselection:`Accounting --> Customers --> Batch Payments` or
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/receivables/customer_payments/recording.rst:84]: To do so, go to :menuselection:`Accounting --> Customers --> Batch Payments` or

File: content/applications/finance/accounting/receivables/customer_payments/recording.rst [hunk 2]
```diff
@@ -94,9 +100,9 @@ Customer Invoices / Vendor Bills`, and click on :guilabel:`⋮` and select :guil
 Matching`, or by going to :menuselection:`Accounting --> Reconciliation`.
 
 .. note::
-   During the reconciliation, if the sum of the debits and credits does not match, it means there is
-   a remaining balance that either needs to be reconciled at a later date or needs to be written off
-   directly.
+   During the :doc:`reconciliation <../../bank/reconciliation/use_cases>`, if the sum of the debits
+   and credits does not match, it means there is a remaining balance that either needs to be
+   reconciled at a later date or needs to be written off directly.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/receivables/customer_payments/recording.rst:105]: During the :doc:`reconciliation <../../bank/reconciliation/use_cases>`, if the sum of the debits
   and credits does not match, there is a remaining balance. This either needs to be
   reconciled at a later date or needs to be written off directly.

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/finance/accounting/receivables/customer_payments/recording.rst [hunk 3]
```diff
@@ -12,10 +12,15 @@ If a payment is **not linked to an invoice or bill**, the customer has an outsta
 your company, or your company has an outstanding debit with a vendor. You can use those outstanding
 amounts to reduce unpaid invoices/bills.
 
+.. seealso::
+   - :doc:`Internal transfers <../../bank/misc/interbank>`
+   - :doc:`../../bank/reconciliation/use_cases`
+   - `Odoo tutorial on Bank Configuration <https://www.odoo.com/slides/slide/bank-configuration-1880?fullscreen=1>`_
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/recording.rst:18]:
```suggestion
   - `Odoo Tutorials: Bank Configuration
     <https://www.odoo.com/slides/slide/bank-configuration-1880>`_
```

For consistency of these links (both title and not full screen)

**Review by jcs-odoo — APPROVED** [review]

**Comment by jcs-odoo** [comment]
I also changed the commit title that was too long (max 70 characters I think) @Donapi 

[IMP] accounting: improve the docpage related to the different ways to record a payment
-->
[IMP] accounting: different ways to record a payment

**Comment by jcs-odoo** [comment]
@robodoo r+

**Comment by jcs-odoo** [comment]
@robodoo retry

**Pull request closed without merging**